toptal/picasso
# fix: remove .prettierignore

[FX-896]

### Description

It was about time we removed .prettierignore. Very low hanging fruit.
The fixes are those that were ignored, but the problem was that `type` from `export type` was previously removed.
This is now fixed. We can safely remove .prettierignore :tada: 

### How to test

- Run this bad boy: 
```yarn prettier packages/picasso-charts/src/index.ts packages/picasso-charts/src/LineChart/index.ts packages/picasso-forms/src/CheckboxGroup/index.ts packages/picasso-forms/src/index.ts packages/picasso-forms/src/RadioGroup/index.ts packages/picasso/src/ApplicationUpdateNotification/index.ts packages/picasso/src/DatePicker/index.ts packages/picasso/src/FormLabel/index.ts packages/picasso/src/Grid/index.ts packages/picasso/src/index.ts packages/picasso/src/MonthSelect/index.ts packages/picasso/src/NativeSelectOptions/index.tsx packages/picasso/src/NativeSelectPlaceholder/index.tsx packages/picasso/src/Select/index.ts packages/picasso/src/TimePicker/index.ts packages/picasso/src/utils/Transitions/index.ts packages/picasso/src/YearSelect/index.ts packages/topkit-analytics-charts/src/AnalyticsChart/index.ts -w ```

There should be no changes to the files, and especially, `type` should not get removed anywhere.

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| Insert screenshots or screen recordings | Insert screenshots or screen recordings |

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>


[FX-896]: https://toptal-core.atlassian.net/browse/FX-896?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
